raiden/raidenwebui/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
@andrevmatos Remove this from here. We don't commit personal editor configurations in the repository.